astropy/io/ascii/setup_package.py

```diff
@@ -72,6 +72,8 @@ def get_package_data():
                                    't/simple3.txt',
                                    't/simple4.txt',
                                    't/simple5.txt',
+                                   't/simple_latin1.txt',
```

Comment:
It looks like your current test strategy is to generate the appropriate file on the fly, so these are not needed.